conans/test/functional/toolchains/cmake/test_cmake_toolchain.py

```diff
@@ -1129,3 +1129,100 @@ def test_cmake_toolchain_vars_when_option_declared():
     t.run_command("cmake -S . -B build/ -DBUILD_SHARED_LIBS=ON")
     assert "mylib target type: SHARED_LIBRARY" in t.out
     assert "mylib position independent code: ON" in t.out
+
+
+#@pytest.mark.tool_cmake()
+def test_find_program_for_tool_requires():
+    """Test that the same reference can be both a tool_requires and a regular requires,
+    and that find_program (executables) and find_package (libraries) find the correct ones
+    when cross building.
+    """
+
+    client = TestClient()
+
+    conanfile = textwrap.dedent("""
+        from conans import ConanFile
+        class TestConan(ConanFile):
+            name = "foobar"
+            version = "1.0"
+            settings = "os", "arch", "compiler", "build_type"
+            exports_sources = "*"
+            def layout(self):
+                pass
+            def package(self):
+                self.copy(pattern="lib*", dst="lib")
```

Comment:
Use modern ``copy()`` for merging to 2.0?

Reply:
Will do! Will update shortly. The test as it is needs other changes to work on 2.0